src/app/contextBroker/contextBroker.cpp

```diff
@@ -330,6 +331,10 @@ PaArgument paArgs[] =
 //
 // /v2 API
 //
+
+#define EPS                EntryPointsRequest
+#define EPS_V2             1, { "v2"             }
```

Comment:
These are the *components* of the request, so, pls change the name to **EPS_COMPS_V2** (you'll see that *all* other defines have COMPS in their name)